src/wazuh_db/wdb_parser.c

```diff
@@ -5804,12 +5805,18 @@ int wdb_parse_global_get_agents_by_connection_status(wdb_t* wdb, char* input, ch
     }
     connection_status = next;
 
+    /* Get limit */
```

Comment:
Get first the `node_name` and if it is not null, get the `limit`.
```c
    /* Get node name */
    next = strtok_r(NULL, delim, &savedptr);
    if (next != NULL) {
        node_name = next;

        /* Get limit */
        next = strtok_r(NULL, delim, &savedptr);
        if (next == NULL) {
            mdebug1("Invalid arguments 'limit' not found.");
            snprintf(output, OS_MAXSTR + 1, "err Invalid arguments 'limit' not found");
            return OS_INVALID;
        }
        limit = atoi(next);
    }
```

Reply:
Done!